### ISSUE TITLE
[Pymode] Found errors and warnings: 0

### ISSUE THREAD

**Issue by raratiru** [python-mode/python-mode#878]
Last week I updated python-mode and the linter ceased working.

I am in develop branch and I have removed all `.pyc` files and `__cache__` directories as per #777 but this time the magic is not happening.

**Comment by raratiru** [comment]
**Update:**

I have just found out that commenting out this line:

`let g:pymode_lint_ignore = 'E501,E123,F0401,C0111,E1002,E1004,C0301,R0201,W0212,W0201'`

The linter worked giving the error:
`error| E0100 SyntaxError: invalid syntax [pylama] `
which is not included in the ignored errors.

Re-enabling the line all errors seem to be ignored.

---

Actually I am using [vim-plug](https://github.com/python-mode/python-mode#using-vim-plug) and I have this line enabled.

When I change to python-mode directory it is already in develop branch. I have just reinstalled it, just in case.
